lib/populator.js

```diff
@@ -24,6 +24,16 @@ function Populator(options) {
   });
 }
 
+// Only adds the dhis version to DHIS URL if it is specified. This ensures backwards compatibility.
+// Needed as DHIS v2.29 api has breaking changes so we need to use the older apis
+function createDhisUrl(dhisVersion, dhisPath) {
+  let tempPath = dhisPath
+  if (dhisVersion) {
+    tempPath = `${dhisVersion}/${tempPath}`
```

Comment:
Instead of using a temporary variable you could return `` `api/${dhisVersion}/${dhisPath}` `` here.